src/client/pages/VergunningDetail/EigenParkeerplaats.tsx

```diff
@@ -0,0 +1,81 @@
+import type { EigenParkeerplaats as EigenParkeerplaatsType } from '../../../server/services';
+import { defaultDateFormat } from '../../../universal/helpers';
+import InfoDetail from '../../components/InfoDetail/InfoDetail';
+import { Location } from './Location';
+
+function getVerzoekType(vergunning: EigenParkeerplaatsType) {
+  switch (true) {
+    case vergunning.isNewRequest:
+      return 'Nieuwe aanvraag';
+    case vergunning.isCarsharingpermit:
+      return 'Autodeelbedrijf';
+    case vergunning.isLicensePlateChange:
+      return 'Kentekenwijziging';
+    case vergunning.isRelocation:
+      return 'Verhuizing';
+    case vergunning.isExtension:
+      return 'Verlenging';
+  }
+}
```

Comment:
Ik zie deze functionaliteit liever in de koppel-api verschijnen. `verzoekType: getVerzoekType()`